pages/example-6-xss/solution-1.tsx

```diff
@@ -0,0 +1,40 @@
+import { BlogPost, getSpecialBlogPosts } from '../../database/blogPosts';
+import { CommonContent } from './common';
+
+type Props =
+  | {
+      error: string;
+    }
+  | {
+      blogPosts: BlogPost[];
+    };
+
+export default function MissingAuthenticationGssp(props: Props) {
```

Comment:
the component name doesn't describe the contents (for other files here too)

I'll take this over

Reply:
Done in c77f4a7